docs/docs/static-vs-normal-queries.md

```diff
@@ -2,35 +2,21 @@
 title: Static vs Normal Queries
 ---
 
-> This documentation isn't up to date with the latest version of Gatsby.
->
-> Outdated areas are:
->
-> - describe how queries are stripped and JSON imports are rewritten
->
-> You can help by making a PR to [update this documentation](https://github.com/gatsbyjs/gatsby/issues/14228).
+Gatsby handles 3 varieties of GraphQL queries: Page queries (sometimes for simplicity referred to as "normal queries"), static queries using the `<StaticQuery />` component, and static queries using the `useStaticQuery` hook.
 
-## How StaticQuery differs from page query
+## Differences between varieties of GraphQL queries
 
-StaticQuery can do most of the things that page query can, including fragments. The main differences are:
+Static queries differ from Gatsby page queries in a number of ways. Gatsby is capable of handling queries with variables for pages because of its awareness of page context while generating them, and **page queries can only be made in top-level page components**.
```

Comment:
```suggestion
Static queries differ from Gatsby page queries in a number of ways. For pages, Gatsby is capable of handling queries with variables because of its awareness of page context. However, **page queries can only be made in top-level page components**.
```

Reply:
Overall, this is really helpful context for understanding the differences.
Feel free to disagree with the edits, it just felt a bit more readable this way.
I also wonder what we can do to better define "page context"?